src/main/modules/session/index.ts

```diff
@@ -9,7 +9,9 @@ const { Logger } = require('@hmcts/nodejs-logging');
 export class Session {
   logger = Logger.getLogger('session');
   enableFor(app: Express): void {
-    const redis = new Redis(config.get<string>('secrets.pcs.redis-connection-string'));
+    const redisConnectionString =
+      process.env.REDIS_CONNECTION_STRING || config.get<string>('secrets.pcs.redis-connection-string');
```

Comment:
this should work as there is env var mapping already?

```suggestion
      config.get<string>('secrets.pcs.redis-connection-string');
```